packages/ontario-design-system-global-styles/src/styles/scss/2-tools/mixins/_form-spacing.mixins.scss

```
@@ -0,0 +1,74 @@
+@use '../../1-variables/spacing.variables' as spacing;
+
+// Mixin for generating individual vertical form spacing classes
+@mixin ontario-vertical-form-spacing($modifier, $spacing-variable) {
+	.ontario-vertical-form-spacing--#{$modifier} {
+		ontario-input:not(:last-child),
+		ontario-checkboxes:not(:last-child),
+		ontario-dropdown-list:not(:last-child),
+		ontario-textarea:not(:last-child),
+		ontario-radio-buttons:not(:last-child),
+		ontario-button:not(:last-child),
+		ontario-date-input:not(:last-child) {
+			margin-bottom: (#{$spacing-variable});
+			display: block;
+		}
+	}
+}
+
+// Mixin for generating nested vertical form spacing classes
+@mixin ontario-nested-vertical-form-spacing($parent-modifier, $parent-spacing, $child-spacings) {
+	.ontario-vertical-form-spacing--#{$parent-modifier} {
+		@each $child-spacing in $child-spacings {
+			$child-modifier: nth($child-spacing, 1);
+			$child-spacing-variable: nth($child-spacing, 2);
+
+			.ontario-vertical-form-spacing--#{$child-modifier} {
+				ontario-input:not(:last-child),
+				ontario-checkboxes:not(:last-child),
+				ontario-dropdown-list:not(:last-child),
+				ontario-textarea:not(:last-child),
+				ontario-radio-buttons:not(:last-child),
+				ontario-button:not(:last-child),
+				ontario-date-input:not(:last-child) {
+					margin-bottom: (#{$child-spacing-variable});
```

We can keep this as a number

```suggestion
					margin-bottom: $child-spacing-variable;
```